Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -304,11 +304,13 @@ void Start()
             gameObject.SetActive(true); // client disables extra cameras, so set it true
             transform.position = new Vector3(0f, 10000000f, 0f); // keep this in a far away place
 
+#if !UNITY_EDITOR
```

Comment:
why?

Reply:
in editor someone might toggle `audioDataToggle` in the inspector and wonder why it isn't working (because the cameras were turned off on start).